script/release

```diff
@@ -39,4 +39,53 @@ git fetch -t origin
 # Tag it and bag it.
 
 gem push github-pages-*.gem && git tag "$tag" &&
-  git push origin master && git push origin "$tag"
+git push origin master && git push origin "$tag"
+
+# create a new release on github
+gh release create "$tag" --notes "Automated release for $tag"
```

Comment:
Is this assuming `gh` is authenticated locally? I think this is a new dependency in the scripts, so we might want to ensure it's documented and verify that `gh` has the appropriate permissions in the script. We can do something like [`gh auth status -h github.com`](https://cli.github.com/manual/gh_auth_status) to verify it's logged in and credentialed for `github.com` before continuing.